examples/selection_tools/polygon_selector_scatter.py

```diff
@@ -0,0 +1,49 @@
+"""
+Polygon Selectors with ScatterGraphic
+=====================================
+
+Example showing how to use a `PolygonSelector` with a scatter plot.
+"""
+
+# test_example = false
```

Comment:
make this `true`, we'd want to have a ground truth screenshot for this since it's a new feature.